backend/src/main/resources/liquibase/create-versioning-tables-changelog.xml

```diff
@@ -158,17 +158,18 @@
         </createTable>
     </changeSet>
     <changeSet author="anand" id="create-commit_parent">
+        <validCheckSum>8:a948978fce54da3ca67c5ec548b19323</validCheckSum>
         <preConditions onFail="MARK_RAN">
             <not>
                 <tableExists tableName="commit_parent"/>
             </not>
         </preConditions>
         <createTable tableName="commit_parent">
-            <column name="parent_hash " type="varchar(64)">
```

Comment:
getting error and failed to come up while column name has space

Reply:
https://vertaai.slack.com/archives/C01RBD7LQ67/p1646969279247349?thread_ts=1646945803.483779&cid=C01RBD7LQ67